.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Thanks you for your thinking about use pretti. However, I'm not sure I want to use it. Specially because of https://github.com/prettier/prettier/issues/4131.